cnd/src/quickcheck.rs

```diff
@@ -50,16 +42,6 @@ impl_arbitrary_for_byte_array!([u8; 16]);
 impl_arbitrary_for_byte_array!([u8; 20]);
```

Comment:
I think you should be able to delete this whole file and remove the quickcheck dependency :)